### PR TITLE
feat: use package version for connect client version

### PR DESCRIPTION
**Pull request by KannarFr**

### CODE REVIEW

File: src/message.rs [hunk 1]
```diff
@@ -638,7 +638,6 @@ mod tests {
 
         {
             let connect = message.command.connect.as_ref().unwrap();
-            assert_eq!(connect.client_version, "2.0.1-incubating");
```
Comment (tisonkun) [inline src/message.rs:641]: Let's update the assertion instead of remove it?

Comment (tisonkun) [reply src/message.rs:641]: 1. follow the same rule of generation
2. doesn't malformed (i.e., the `env!("CARGO_PKG_VERSION")` part returns not empty or even in pattern `X.Y.Z`)

Comment (KannarFr) [reply src/message.rs:641]: The assertion is based on the bytes array input of a message which we won't update for each version of pulsar-rs? WDYT?

Not sure to understand your second comment?

Comment (tisonkun) [reply src/message.rs:641]: Make sense. I push one commit to update the bytes so that we at least sync once with the new change.

Comment (KannarFr) [reply src/message.rs:641]: LGTM. Ty!

**Review by tisonkun — APPROVED** [review]
Merging...

Thanks for your contribution @KannarFr!

**Pull request merged**